### PR TITLE
Update tokenizers library version to GA version 1.0.0

### PR DESCRIPTION
**Pull request by tarekgh**

### CODE REVIEW

**Comment by tarekgh** [comment]
CC @stephentoub

@dluc could you please review this change and let me know if you have any questions? Thanks!

**Review by stephentoub — APPROVED** [review]

File: extensions/OpenAI/OpenAI/OpenAI.csproj [hunk 1]
```diff
@@ -15,7 +15,9 @@
 
     <ItemGroup>
         <PackageReference Include="Microsoft.SemanticKernel.Connectors.OpenAI" />
-        <PackageReference Include="Microsoft.ML.Tokenizers" /> <!-- prerelease dependency (NU5104) -->
+        <PackageReference Include="Microsoft.ML.Tokenizers.Data.Cl100kBase" />
+        <PackageReference Include="Microsoft.ML.Tokenizers.Data.O200kBase" />
+        <PackageReference Include="Microsoft.ML.Tokenizers.Data.P50kBase" />
```
Comment (dluc) [inline extensions/OpenAI/OpenAI/OpenAI.csproj:20]:
```suggestion
```

Actually these can be removed, the dependency was not used. All tokenizers are imported via Tiktoken.csproj

File: service/Core/Core.csproj [hunk 2]
```diff
@@ -19,7 +19,9 @@
         <PackageReference Include="HtmlAgilityPack" />
         <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
         <PackageReference Include="Microsoft.Extensions.Http" />
-        <PackageReference Include="Microsoft.ML.Tokenizers" /> <!-- prerelease dependency (NU5104) -->
+        <PackageReference Include="Microsoft.ML.Tokenizers.Data.Cl100kBase" />
+        <PackageReference Include="Microsoft.ML.Tokenizers.Data.O200kBase" />
+        <PackageReference Include="Microsoft.ML.Tokenizers.Data.P50kBase" />
```
Comment (dluc) [inline service/Core/Core.csproj:24]:
```suggestion
```

same here, safe to remove (left over from earlier PR)

**Review by dluc — APPROVED** [review]

**Comment by dluc** [comment]
Thank you @tarekgh 👍

**Pull request merged**